scripts/create_ts_for_integration_tests.py

```diff
@@ -52,9 +53,14 @@
     NodeId(space="PySDK-DMS-time-series-integration-test", external_id="PYSDK integration test 127: clone of 121"),
 ]
 
+NAMES_BY_NUMBER = {
+    int(name.split(":", maxsplit=1)[0].removeprefix("PYSDK integration test ")): name
+    for name in NAMES + SPARSE_NAMES + [node_id.external_id for node_id in NAMES_USING_INSTANCE_ID]
+}
```

Comment:
I don't see the need for this, if its just for the "clone" part. I think that `re.findall++` should be removed in favor of just hardcoding the identifiers.

I can push a commit if you want.